base/Cargo.toml

```diff
@@ -15,4 +15,11 @@ chrono = "0.4"
 derive_deref = "1.1"
 num = "0.4"
 serde = { version = "1", features = ["derive"] }
-thiserror = "1"
\ No newline at end of file
+thiserror = "1"
+
+plonk_protocol_dlog = { git = "https://github.com/o1-labs/proof-systems" }
```

Comment:
need to whitelist 'o1-labs' in deny.toml after #79 is merged

Reply:
Good to know